app/components/info-cards/song-card.tsx

```
@@ -0,0 +1,32 @@
+    // }
+
+    if (currentSong?.id === props.songID && isPlaying) {
+      pauseSong();
+      return;
+    }
+
+    const song = {
+      id: props.songID!,
+      duration: props.duration!,
+      title: props.title!,
+      artists: props.artists!.map(artist => ({ artist })),
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)